apps/wallet/src/shared/constants.ts

```diff
@@ -14,3 +14,6 @@ export const DEFAULT_NFT_IMAGE =
 
 // number of epochs before earning
 export const NUM_OF_EPOCH_BEFORE_EARNING = 2;
+export const UNSTAKE_REQUEST_EVENT_TYPE =
```

Comment:
Should we move these to the core SDK, with a note to keep it in sync with the move counter-parts??